src/@types/topic.ts

```diff
@@ -60,4 +60,4 @@ export type MTopicEvent = EitherAnd<{ [M_TOPIC.name]: MTopicContent }, { [M_TOPI
 /**
  * The event content for an m.room.topic event
  */
-export type MRoomTopicEventContent = { topic: string } & MTopicEvent;
+export type MRoomTopicEventContent = { topic: string | null | undefined } & (MTopicEvent | {});
```

Comment:
This allows any type to be set... https://www.totaltypescript.com/the-empty-object-type-in-typescript

I swear I left a comment but think it was on the commit so may have been lost. @Half-Shot can you please resolve this. This allows a value of `123` as the content which would error on the API for example.

Reply:
![image](https://github.com/user-attachments/assets/227cef85-025c-4e2e-b750-0577f76b60dd)

Reply:
Sure, I'll do a followup now.

Reply:
http://localhost:8008/_synapse/admin/v1/users/@test_deactivated:beefy/media?order_by=created_ts&dir=b&limit=1